scripts/update_robots.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2021 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -eu
+
+repo_root="$(dirname "$0")/.."
+docs_static_path="docs/_static"
+robots_template_path="${repo_root}/${docs_static_path}/robots.txt.template"
+robots_txt_path="${repo_root}/${docs_static_path}/robots.txt"
+latest_version=$("${repo_root}"/scripts/get_version.sh)
+template_line_match="Allow:[[:space:]]/en/v\{\{[[:space:]]LATEST_VERSION[[:space:]]\}\}/"
```

Comment:
I think that it's fine to match just ``{{ LATEST_VERSION }}``, without requiring the `Allow:` part to be there. This way it will work more like you would expect a template to work.

And then I would replace only that part.